test/defaultvaluechanged/expected.json

```diff
@@ -1,59 +1,62 @@
 {
  "crdValidation": [
```

Comment:
It seems the order here is non-determinisitic which is leading to re-ordering of lots of the output? Can that be fixed?

Reply:
I thought most serialization libraries in Go sorted lists on output, but I could be wrong. Being non deterministic in this case seems particularly problematic as it's creating unnecessary churn isn't it?